backend/graph-proxy/src/graphql/workflows.rs

```diff
@@ -1342,4 +1394,83 @@ mod tests {
         });
         assert_eq!(resp.data.into_json().unwrap(), expected_data);
     }
+    #[tokio::test]
+    async fn multiple_workflows_query_with_filter() {
+        let visit = Visit {
+            proposal_code: "mg".to_string(),
+            proposal_number: 36964,
+            number: 1,
+        };
+
+        let mut server = mockito::Server::new_async().await;
+        let mut workflows_response_file_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        workflows_response_file_path.push("test-assets");
+        workflows_response_file_path.push("get-workflows.json");
+
+        let workflows_endpoint = server
+            .mock("GET", &format!("/api/v1/workflows/{}", visit)[..])
+            .match_query(mockito::Matcher::AllOf(vec![mockito::Matcher::UrlEncoded(
+                "listOptions.labelSelector".to_string(),
+                "workflows.argoproj.io/phase in (Error)".to_string(),
+            )]))
+            .with_status(200)
+            .with_header("content-type", "application/json")
+            .with_body_from_file(&workflows_response_file_path)
+            .create_async()
+            .await;
+
+        let argo_server_url = Url::parse(&server.url()).unwrap();
+        let schema = root_schema_builder()
+            .data(ArgoServerUrl(argo_server_url.clone()))
+            .data(None::<Authorization<Bearer>>)
+            .finish();
+
+        let query = format!(
+            r#"
+        query {{
+            workflows(
+                visit: {{proposalCode: "{}", proposalNumber: {}, number: {}}}, 
+                filter: {{ error: true }}
+            ) {{
+                nodes {{
+                    name
+                }}
+            }}
+        }}
+    "#,
```

Comment:
Nitpick: the indentation level of the code starting with `query` in the raw string literal looks a bit odd to me.

I know `rustfmt` doesn't fiddle with code involving raw string literals much so I could see how this doesn't get picked up by the code formatter, but personally, I'd prefer it indented so that code within a function doesn't go to the left as if it's not in the scope of the function (which the last line of the raw string literal does), if you see what I mean.